packages/lib/src/vis/line/GlyphMaterial.tsx

```diff
@@ -33,9 +33,12 @@ const GLYPH_SHADERS = {
   `,
   [GlyphType.Circle]: `
     float alphaSymbol(vec2 coord, float size) {
-      float radius = 0.5;
       float r = distance(coord, vec2(0.5, 0.5));
-      return clamp(size * (radius - r), 0.0, 1.0);
+      if( r < 0.5) {
+        return 1.0;
+      } else {
```

Comment:
You don't need the `else` technically.

Reply:
True, This is more to comply with the syntax of other shaders from silx.